pkg/commands/execution_context.go

```diff
@@ -1143,7 +1146,10 @@ func (ctx *executionContext) monitorCluster(context context.Context, ci *cluster
 		err := ci.instance.CheckIsAlive()
 		if err != nil {
 			logrus.Errorf("Failed to interact with %s: %v", ci.id, err)
-			ctx.destroyCluster(ci, true, false)
+			// TODO what should we do about this error?
```

Comment:
I really don't know what the right thing to do with the error here is, so logged it. If that's the right thing, I will remove the TODO comment and repush, just let me know.

Reply:
Error is logged inside, I will fix this with next PR.